content/dogepedia/articles/dogecoin-folding-at-home.md

```diff
@@ -5,8 +5,8 @@ date = "2021-10-22"
   name = "Dogecoin"
 +++
 
-Dogecoin Folding at Home is a community-based distributed-computing charitable project that utilises participants’ idle computer power to help scientists who are studying Covid, Alzheimer's, Huntington's, Parkinson's, cancers, and more., as they work towards finding cures.  
+Dogecoin Folding at Home is a charitable community-based distributed-computing project that utilises participants’ idle computer power to help scientists who are studying COVID-19, Alzheimer's, Huntington's, Parkinson's, cancers and other diseases as they work towards finding cures.
 
 It uses a very simple GUI interface, and runs in the background using idle computer time via GPUs, CPUs, and ARM processors. Your computer will receive “work units”, and then sends the completed work units back to the FAH servers. These are then compiled with other work units into an overall statistical simulation pertaining to one or more diseases and their potential therapies.   
 
-In addition to helping scientists develop new therapeutics for a variety of diseases, participants are also rewarded with weekly payouts in Dogecoin, which are provided to the organisation by long term sponsors and user donations. Dogecoin Folding at Home supports Windows, Linux, and MacOS operating systems, and to get started or learn more: [click here] (https://www.dogecoinfah.com/index.html)! 
\ No newline at end of file
+In addition to helping scientists develop new therapeutics for a variety of diseases, participants are also rewarded with weekly payouts in Dogecoin, which are provided to the organisation by long term sponsors and user donations. Dogecoin Folding at Home supports Windows, Linux, and MacOS operating systems. To get started or learn more: [click here] (https://www.dogecoinfah.com/index.html)! 
```

Comment:
just need to remove the space in between `[click here] (https://www.dogecoinfah.com/index.html)` and it looks good to go!

Reply:
What space do you mean @xanimo? Ready to change now :)